src/Paket.Core/Cultures.fs

```diff
@@ -0,0 +1,19 @@
+﻿namespace Paket
+open System
+open System.Collections.Generic
+open System.Globalization
+
+[<CompilationRepresentation (CompilationRepresentationFlags.ModuleSuffix)>]
```

Comment:
I copy pasted them from the [Cache module](https://github.com/fsprojects/Paket/blob/master/src/Paket.Core/Cache.fs#L48), no idea what they mean.